src/main/java/sirius/web/http/TunnelHandler.java

```diff
@@ -62,6 +64,11 @@
     private final IntConsumer failureHandler;
     private final CallContext cc;
     private final Watch watch;
+    private volatile long timeToDns;
+    private volatile long timeToConnectAttept;
```

Comment:
```suggestion
    private volatile long timeToConnectAttempt;
```

Reply:
Will be fixed with the update to the next async http client release.